src/domains/tasks/CreateTask/types.tsx

```diff
@@ -34,21 +34,33 @@ export const TaskTimeEstimateSchema = z.enum([
 
 export const CreateTaskFormSchema = z.object({
   category: TaskCategorySchema.default("Cleaning"),
-  title: z.string().min(10),
-  details: z.string().min(25),
-  budget: z.number().gt(5),
+  title: z
+    .string({
+      required_error: "Title is required",
+    })
+    .min(10, { message: "Title must be at least 10 characters long" }),
+  details: z
+    .string({ required_error: "Details are required" })
+    .min(25, { message: "Details must be at least 25 characters long" }),
+  budget: z.coerce
+    .number({
+      invalid_type_error: "Budget must be a number",
+    })
+    .gt(5, { message: "Budget must be at least $5" }),
   timeOfArrival: TaskTimeOfArrivalSchema.default("7am-10am"),
   timeEstimate: TaskTimeEstimateSchema.default("1-3hrs"),
   status: TaskStatusSchema.default("open"),
-  dueDate: z
-    .date()
+  dueDate: z.coerce
```

Comment:
This was an interesting one because dates from the inputs were `2023-01-26` and were failing input validation. I had to add the `z.coerce` to get this to work.

Commenting in case I need to come back to this in the future.